src/function.js

```diff
@@ -5,12 +5,9 @@ export const saveData = (list) => {
 export const setCompleted = (todoList, box) => {
```

Comment:
- [ ] I would highly recommend naming the files in a meaningful way, that will help another developer and even recruiters to navigate through your project 😄 
    - For example, this file is named `function` that is too vague, kindly name this and other files with meaningful names.

Reply:
Thank you for your feedback, all changes have been implemented